webapp/packages/core-blocks/src/Containers/BASE_CONTAINERS_STYLES.ts

```diff
@@ -73,29 +82,31 @@ export const BASE_CONTAINERS_STYLES = composes(
       margin: 10px;
       padding: 24px;
       border-radius: 4px;
-
       &[form] {
         padding-right: 30%;
       }
     }
 
-    GroupItem {
-      min-width: min-content;
-    }
-
-    GroupTitle {
-      composes: theme-typography--body1 from global;
-      font-weight: 400;
-      margin: 0;
-      text-transform: uppercase;
-      opacity: 0.9;
+    Container, ColoredContainer {
+      & > [flexItemGrow] {
+        flex-grow: 1;
+      }
+      & > [flexItemTiny] {
+        flex-basis: 100px;
+      }
+      & > [flexItemSmall] {
+        flex-basis: 182px;
+      }
+      & > [flexItemMedium] {
+        flex-basis: 300px;
+      }
+      & > [flexItemLarge] {
+        flex-basis: 400px;
+      }
```

Comment:
`flex-grow: 1` should be by default, sizes looks strange, large-item should be bigger at least